.vale/styles/AsciiDoc/LinkContainsLinkText.yml

```diff
@@ -0,0 +1,10 @@
+---
+extends: existence
+scope: raw
+level: warning
+link: https://redhat-documentation.github.io/supplementary-style-guide/#cloud-services-links-hypertext
+message: "Link is missing accessibility descriptive link text."
+raw:
+  - 'link:.*\[\]'
+  - 'link:.*\[[cC]lick here\]'
+  
```

Comment:
```suggestion

```

Reply:
Is this suggested to remove a trailing newline?

Reply:
You had 2 spaces and a new line :sweat_smile:

A nit - my yaml linter reported it. Usually not a problem.